src/environment_react_native.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+var DeviceEventEmitter = require('react-native').DeviceEventEmitter;
+var WebRTC = require('react-native-webrtc');
```

Comment:
yes, if you are running on react native, it will provide:

* react-native module
* global var
* global WebSocket and Promise
* the rest except I commented as `null`

`react-native-webrtc` is a third party module needs to install manually.

the field `react-native` in package.json is specifically for react native, it's packager will check that field and replace files like `browser` field does.